kensu/google/cloud/bigquery/job/bigquery_stats.py

```diff
@@ -1,16 +1,18 @@
+import decimal
 import logging
 import datetime
+logger = logging.getLogger(__name__)
 
 from kensu.google.cloud.bigquery import Client
 from kensu.utils.kensu_provider import KensuProvider
 
 
-def compute_bigquery_stats(table_ref, table, client, stats_aggs, input_filters=None):
+def compute_bigquery_stats(table_ref=None, table=None, client=None, stats_aggs=None, input_filters=None, query=None):
```

Comment:
it's good practice to inform when a value is required 
if None == query == table_ref:
    raise Exception("compute_bigquery_stats requires either a table_ref or a query")